src/components/Dashboard.jsx

```diff
@@ -288,22 +289,12 @@ const RestaurantDashboard = () => {
             </Text>
             <BarChartCustom
                 data={chartData}
-                title='Total CO2 emissions of meals'
+                title={t('CO2_TOTAL')}
```

Comment:
CO2 should be the mean CO2 values and not total

Reply:
fix'd in commit [82ccf611](https://github.com/ohturuokasovellus/OhtuRuokasovellus/pull/88/commits/82ccf611caa080de096993ede6c3fd70aae33dba) 🤝